larray/viewer/start.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+from larray.viewer.api import edit
+
+def main():
+    if os.name == 'nt':
```

Comment:
check that sys.stdout is None instead? so that we do not redirect to null needlessly?

(I admit it might not be worth changing given the intent of the script)